### ISSUE TITLE
Video was not recorded upon assertion failure expect true to be false

### ISSUE THREAD

**Issue by PFaz540** [presidenten/wdio-video-reporter#9]
**Describe the bug**
- A video was not output when I purposely failed a test.
- If I had the below code in my test, a video would be output as expected:
```
const currentURL = browser.getUrl();
expect(currentURL, "User was not re-directed to '/my-account'").to.contain("/my-accountZZZZZZZ");
```
- However, if I had the below code - the test would obviously fail but no video is output:
```
expect(true).to.be.false;
```

The actual assertion I was using in my test that flagged the issue to me was:
```
it("confirms the user can 'Proceed To Checkout'", function() {
         const breadcrumbsYourDetails = $("[href='/checkout/details'].selected")
         expect(breadcrumbsYourDetails.isDisplayed()).to.be.false; // FAILING ON PURPOSE
      });
```

**Log**
https://pastebin.com/TwRQQq0W

**To Reproduce**

Spec file:
```
const { expect } = require("chai");

describe("Bug for wdio video reporter", function() {
    it("fails on purpose", function() {
        expect(true).to.be.false;
    });
});
```

The 'reporters' array in my config file:
```
reporters: [
        "spec",
        [ video , {
            outputDir: "./videos",
            saveAllVideos: false, // If true, also saves videos for successful test cases
            videoSlowdownMultiplier: 3, // Higher to get slower videos, lower for faster videos [Value 1-100]
            videoRenderTimeout: 5 // Max seconds to wait for a video to finish rendering
        }]
    ],
```

**Expected behavior**
- I'd expect an .mp4 file to be output upon assertion/chai failures

**Environment (please complete the following information):**
 - **wdio-video-reporter version:** 1.3.1
 - **WebdriverIO version:** 5.8.3
 - **Mode:** WDIO Testrunner
 - **If WDIO Testrunner, running sync/async:** sync
 - **Node.js version:** 12.1.0
 - **NPM version:** 6.9.0
 - **Browser name and version:** Chrome
 - **Platform name and version:** Mac Mojave 10.14.4

**Desktop (please complete the following information):**
 - OS: MAC OS
 - Browser Chrome
 - Version 74.0.3729.157

**Comment by presidenten** [comment]
Thanks for the extensive bug report. It was really helpful to isolate the bug.
There seems to be an issue when videos have very few frames.

I will release a fix shortly.

**Comment by presidenten** [comment]
Issue is fixed in [v1.4.0](https://github.com/presidenten/wdio-video-reporter/releases/tag/v1.4.0)
